### PR TITLE
feat(manager/web): chunked invite flow - phase zero

### PR DESCRIPTION
**Pull request by daniellacosse**
I've implemented what I think is the smallest possible usable change as a "Phase Zero". Clicking on the share button simply opens the following dialog:

Screen Shot 2022-10-11 at 15 38 33

### CODE REVIEW

**Review by kevindamm-jigsaw — APPROVED** [review]

**Review by fortuna — APPROVED** [review]

File: src/server_manager/web_app/ui_components/outline-share-dialog.ts [hunk 1]
```diff
@@ -111,33 +118,31 @@ Polymer({
...
-        <p>[[localize('share-invite')]]</p>
+      <div
+        contenteditable=""
+        id="selectableInviteText"
+        style="-webkit-text-size-adjust: 100%;"
+        inner-h-t-m-l="[[localize('share-invite-html')]]"
+      ></div>
+      <div id="copyInvitationIndicator" hidden="">[[localize('share-invite-copied')]]</div>
+      <div class="button-row">
+        <paper-button id="copyButton" on-tap="copyInvite"
```
Comment (fortuna) [inline src/server_manager/web_app/ui_components/outline-share-dialog.ts:129]: Perhaps we should rename "invite/invitation" to "instructions"?
The invitation is not really complete without the access key.

File: src/server_manager/web_app/ui_components/outline-share-dialog.ts [hunk 2]
```diff
@@ -111,33 +118,31 @@ Polymer({
...
-          <li>
-            [[localize('share-invite-instructions')]]
-            <a
-              href="https://github.com/Jigsaw-Code/outline-client/blob/master/docs/invitation_instructions.md"
-              >https://github.com/Jigsaw-Code/outline-client/blob/master/docs/invitation_instructions.md</a
-            >
-          </li>
-        </ol>
+      <div contenteditable="" id="selectableAccessKey" style="-webkit-text-size-adjust: 100%;">
+        <pre><a href="[[accessKey]]">{{accessKey}}</a></pre>
```
Comment (fortuna) [inline src/server_manager/web_app/ui_components/outline-share-dialog.ts:135]: No need to use two-way binding here.
```suggestion
        <pre><a href="[[accessKey]]">[[accessKey]]</a></pre>
```

**Review by fortuna — COMMENTED** [review]
Looks good. Some minor comments

File: src/server_manager/messages/en.json [hunk 3]
```diff
@@ -262,10 +262,11 @@
...
+  "share-invite-copied": "Copied invitation to clipboard",
   "share-invite-copy": "Copy invitation",
```
[the remark this answers is not in the document]
Comment (daniellacosse) [reply src/server_manager/messages/en.json:268]: This is to the spec I was given and will require us to regenerate the translations: https://www.figma.com/file/vF9xxkzD6D49tEaqpH9Sbf/Outline-Updates?node-id=99%3A66

If you don't mind, I think we should merge this and discuss further changes with UX

**Review by fortuna — APPROVED** [review]
Thanks for getting the new flow done!

**Pull request merged**